README.md

```diff
@@ -55,15 +54,14 @@ def test_line_label(sample_data, cleanup_files):
         .add(
             so.Lines(),
             so.Agg(),
-            sor.Rolling(window_type="gaussian", window_kwargs={"std": 2}),
+            rolling := sor.Rolling(window_type="gaussian", window_kwargs={"std": 2}),
             legend=False,
         )
-        # This will be much easier when compound marks are implemented.
-        # Will be able to do so.Line() + LineLabel().
+        
         .add(
             sor.LineLabel(offset=5),
             so.Agg(),
-            sor.Rolling(window_type="gaussian", window_kwargs={"std": 2}),
+            rolling,
             legend=False,
         )
         .add(so.Band(), so.Est(errorbar="se"), legend=False)
```

Comment:
I don't see a band in the figure below — is there supposed to be one, and if not, what's this line doing?

Reply:
I agree, I think I took it out in one of my actual testing but forgot it here. I'll take it out.